core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -34,6 +34,37 @@
 #include <sm/teesmc_opteed_macros.h>
 #include <sm/teesmc_opteed.h>
 
+.section .data
+.balign 4
+
+#ifdef CFG_BOOT_SYNC_CPU
```

Comment:
Looks like there is a dead-lock if I set `CFG_BOOT_SYNC_CPU=y` on vexpress. I think it comes from the way the boot of each cpu is performed (all at once, or one following another one)